### DISCUSSION TITLE
Supabase can't send emails with custom SMTP

### DISCUSSION THREAD

**Discussion by testdgb** [supabase/supabase#17805] [Questions]
Hey there,
I have set up my self-hosted Supabase database, so far it’s going great but there’s an issue that I can’t figure out. I can’t send authentication emails at all, every time I try (whether creating a new user, sending password recovery, etc.) I get an error on the Supabase Studio saying that it’s unable to process the request. 

I have set up the credentials for my SMTP provider on both .env and docker-compose.yml. Yet, it still doesn’t work. The credentials are 100% correct. 

I’m using Brevo SMTP, and I host the supabase on Linode Ubuntu server. 

Thanks!!

**Comment by silentworks** [comment]
This is likely something wrong in your setup. I have Brevo SMTP working in my hosted Supabase instance which under the hood uses the same method of sending custom SMTP. You should provide more information here as to which env variables you have set and if you restarted your docker services after setting this information or not.
